ldap/servers/slapd/back-ldbm/db-bdb/bdb_layer.c

```diff
@@ -1914,18 +1924,8 @@ bdb_get_aux_id2entry_ext(backend *be, DB **ppDB, DB_ENV **ppEnv, char **path, in
         }
         *ppEnv = mypEnv->bdb_DB_ENV;
     }
-    rval = db_create(&dbp, *ppEnv, 0);
+    rval = dbbdb_create_db_for_open(be, "dblayer_get_aux_id2entry_ext", dbflags, &dbp, *ppEnv);
```

Comment:
Is it a typo in funcname argument? You have "dblayer_get_aux_id2entry_ext" but the function name it's called from is bdb_get_aux_id2entry_ext.

Reply:
Not really a typo: for consistency I used the name that is used in all slapi_log_xxx calls in this function.
( It is an historical thing: when Ludwig split the dblayer code, he has chosen to keep the original function names in the log. I just used the same logic and reuse the same  old name.
IMHO we should keep the same name than the other logs within same function.
Now whether we should keep the old name or use the current name everywhere could be argued !

Reply:
> ( It is an historical thing: when Ludwig split the dblayer code, he has chosen to keep the original function names in the log. I just used the same logic and reuse the same old name.

Oh, I didn't know there's a historical reasoning there (and I missed that other slapi_log functions do the same).

> IMHO we should keep the same name than the other logs within same function.

For sure!

> Now whether we should keep the old name or use the current name everywhere could be argued !

I think it'll be nice to bring more consistency there because we have:

- `dblayer_xxx` functions with `dblayer_xxx`  log function strings
- `bdb_xxx` functions with `bdb_xxx`
- `bdb_xxx` functions with `dblayer_xxx` 
- `bdb_xxx` functions without `bdb` prefix in the log function string (i.e. `bdb_check_and_set_import_cache` with `check_and_set_import_cache`)

Maybe more; I haven't checked the whole file.
I think the inconsistency is a bit confusing, but it's clearly a task for a separate PR. :)

You have my ack for this one! Thanks!